CHANGELOG.md

```diff
@@ -1,5 +1,92 @@
 # Changelog
 
+## Pelion Edge 2.2 - January 2021
```

Comment:
```suggestion
## Pelion Edge 2.2 - February 2021
```

Not sure if this needs to be changed now.

Reply:
Sure. Added this to the next commit.